index.html

```diff
@@ -10,7 +10,7 @@
     <link rel="stylesheet" href="style.css">
```

Comment:
The head should have a title tag in it to give a title to your website - right now your website is just called index.html, which doesn't look very good.